packages/grpc-native-core/index.d.ts

```diff
@@ -202,7 +204,7 @@ declare module "grpc" {
      * @param implementation Map of method names to method implementation
      * for the provided service.
      */
-    addProtoService(service: Service | ServiceDefinition, implementation: { [name: string]: handleCall }): void;
+    addProtoService<T>(service: Service<T> | ServiceDefinition, implementation: T): void;
```

Comment:
I have the same concerns here as with `addService` regarding the usefulness of the type parameterization.

Reply:
See above